### DISCUSSION TITLE
Layout does not work on external windows

### DISCUSSION THREAD

**Discussion by feedhopper** [TfTHacker/DashboardPlusPlus#9] [General]
Hi there,

THANKS for this great mod of Obsidian pages. I use it every day and I LOVE IT!!!

One thing, though I just realised: the layout gets juggled about when the pages are displayed on an external window.
Can you fix this? It would be SO GREAT!!

Thank you!!!

**Comment by TfTHacker** [comment]
Thank you for reporting this. I did some testing, and it turns out it's a known bug with the banner plugin. Hopefully, it will get fixed. If you remove the banner, the page will be more readable, but it stinks.

More info:
https://github.com/noatpad/obsidian-banners/issues/67
